certbot-apache/certbot_apache/_internal/parser.py

```diff
@@ -355,7 +357,7 @@ def filter_args_num(self, matches: str, args: int) -> List[str]:
 
         return filtered
 
-    def add_dir_to_ifmodssl(self, aug_conf_path: str, directive: str, args: List[str]) -> None:
+    def add_dir_to_ifmodssl(self, aug_conf_path: str, directive: str, args: Sequence[str]) -> None:
```

Comment:
Given the unfortunate way `Sequence[str]` interacts with `str`, let's revert this back to `List`?

```suggestion
    def add_dir_to_ifmodssl(self, aug_conf_path: str, directive: str, args: List[str]) -> None:
```

Reply:
Sorry I missed that one. And since we are there, I replace all references of `Sequence[str]` by `List[str]` in the project, not only the apache module.